firmware/console/binary/tunerstudio.cpp

```diff
@@ -277,6 +277,28 @@ static void onlineApplyWorkingCopyBytes(int currentPageId, uint32_t offset, int
 	}
 }
 
+/**
+ * Read internal structure for Live Doc
+ * This is somewhat similar to read page and somewhat similar to read outputs
+ * We can later consider combining this functionality
+ */
+static void handleGetStructContent(ts_channel_s *tsChannel, int structId, int size) {
+	tsState.readPageCommandsCounter++;
+
+	const void *addr = NULL;
+	if (structId == LDS_CLT_INDEX) {
+		addr = &engine->engineState.cltCurve;
```

Comment:
the type-safe way to do this is
`addr = static_cast<thermistor_state_s*>(&engine->engineState.cltCurve)`
That cast will automagically fix any vtable-related woes.

Reply:
Done, thank you!